### PR TITLE
Replace msrestazure with azure.core

### PR DESCRIPTION
**Pull request by Jing-song**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

databricks
import-export
internet-analyzer
peering
db-up
virtual-network-tap

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/peering/setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '0.2.1'
+VERSION = '1.0.0'
```
Comment (AllyW) [inline src/peering/setup.py:15]: @Jing-song the initial version for extension should be 1.0.0b1, or 1.0.0 if previous version is stable version

Comment (AllyW) [reply src/peering/setup.py:15]: please see the doc here: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#notes-1

Comment (Jing-song) [reply src/peering/setup.py:15]: Is `0.2.1` a preview version? Do I need to change it to `1.0.0b1`?

**Comment by kairu-ms** [comment]
Hi @wangzelin007, can our release pipeline support to release multi extensions in one PR?

**Review by jsntcy — APPROVED** [review]

**Comment by wangzelin007** [comment]
@kairu-ms @jsntcy 
Yes, Extension release already supports releasing multiple extensions in one PR.

**Review by wangzelin007 — APPROVED** [review]

**Pull request merged**